### ISSUE TITLE
How to add new template without restartconfd?

### ISSUE THREAD

**Issue by zhousoft** [kelseyhightower/confd#588]
Hello, is there any method to add new template without restart confd?
I already had a template like that:
```
 [template]
   src = "myapp_nginx.tmpl"      
   dest = "/myapp/nginx.conf"     
   keys = [  
         "/myapp/nginxconf",  
  ]  
  reload_cmd = "/usr/local/nginx/sbin/nginx -s reload"    
```

when I add a new template,I have to restart confd to make the new template works,but the confd will reload my nginx also, is there any method to make confd load the new template without restart?

**Comment by okushchenko** [comment]
No, there is no way to reload reconfiguration other than a restart of confd process. I think that a reload of config files on SIGHUP is generally a good idea and a nice feature to have. Feel free to open a PR!

**Comment by zhousoft** [comment]
Thanks for your reply, actually I am not a Go programmer, but i'm glad to have a try.
